parametric_study.py

```diff
@@ -233,8 +233,11 @@ def create_plots(outdir: str, outfile: str):
     speed = df["Speed"]
     progress = tqdm.tqdm(desc="Parametric sweep", total=len(speed))
     for omega in speed:
+        petsc_options = {"ksp_type": "cg", "pc_type": "lu"}
```

Comment:
A bit unusual to combine CG and LU. Perhaps the ksp type should be 'preonly'?

Reply:
Sure, I was just playing around with solver setup's earlier today, as the options supplied by RR in the template models are highly inefficient.